examples/test_examples.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -e
+
+gpu=${1:--1}
+echo Using gpu ${gpu}
+
+# Tox21
+echo --- Testing Tox21 ---
+cd tox21 && ./test_tox21.sh ${gpu} && cd ..
```

Comment:
Personally, I like `bash -x test_tox21.sh` because when the test fails, we can identify where is the problem. How do you think?

Reply:
sure